client/wordpress-com.js

```diff
@@ -341,7 +341,14 @@ sections.push( {
 	module: 'login',
 	enableLoggedOut: true,
 	secondary: false,
-	isomorphic: true
+	isomorphic: {
+		cache: {
+			queryParams: {
+				include: [ 'client_id' ],
+				exclude: [ 'redirect_to' ],
+			}
```

Comment:
Ditto @ockham's take above. Removing this whitelist/blacklist functionality would lead to simpler code as well.